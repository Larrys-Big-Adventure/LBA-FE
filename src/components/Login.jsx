import React from 'react';

const Login = () => {
  return <div>Login here bruh</div>;
};

export default Login;
