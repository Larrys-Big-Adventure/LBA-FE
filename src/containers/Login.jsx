import React from 'react';
import styled from 'styled-components';
import { LoginForm } from '../components';

const Login = () => {
  return (
    <LoginContainer>
      <LoginForm />
    </LoginContainer>
  );
};

const LoginContainer = styled.div`
  margin: 0 auto;
`;

export default Login;
