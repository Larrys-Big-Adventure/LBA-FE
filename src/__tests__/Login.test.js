import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { cleanup } from '@testing-library/react';
import render from '../__testHelpers__/render';

import Login from '../components/Login.jsx';

afterEach(cleanup);

describe('Login', () => {
  it('renders without crashing', () => {
    render(<Login />);
  });
});
