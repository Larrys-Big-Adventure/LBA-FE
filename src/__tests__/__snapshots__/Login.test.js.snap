// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login matches previous snapshot 1`] = `
<div
  className="sc-bdVaJa dvAieK"
>
  <form
    action=""
  >
    <input
      type="text"
    />
    <button
      type="submit"
    >
      Submit
    </button>
  </form>
</div>
`;
